src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/UITestRunner.java

```diff
@@ -35,47 +39,118 @@
  */
 public class UITestRunner {
     private static final int defaultPort = 8580;
-    private static RemoteRobot robot = null;
+    private static RemoteRobot remoteRobot = null;
     private static Process ideProcess;
-
-    public static RemoteRobot runIde(String ideaVersion, int port) {
+    private static int intelliJIdeaVersion;
+
+    /**
+     * Start the given version of IntelliJ Idea listening on the given port
+     *
+     * @param ideaVersion version of the IntelliJ Idea to start
+     * @param port        port number on which will the IntelliJ Idea be listening
+     */
+    public static RemoteRobot runIde(IdeaVersion ideaVersion, int port) {
         makeSureAllTermsAndConditionsAreAccepted();
 
         String osName = System.getProperty("os.name").toLowerCase();
         ProcessBuilder pb;
         if (osName.contains("windows")) {
-            pb = new ProcessBuilder(".\\gradlew.bat", "runIdeForUiTests", "-PideaVersion=" + ideaVersion, "-Drobot-server.port=" + port);
+            pb = new ProcessBuilder(".\\gradlew.bat", "runIdeForUiTests", "-PideaVersion=" + ideaVersion.toString(), "-Drobot-server.port=" + port);
         } else {
-            pb = new ProcessBuilder("./gradlew", "runIdeForUiTests", "-PideaVersion=" + ideaVersion, "-Drobot-server.port=" + port);
+            pb = new ProcessBuilder("./gradlew", "runIdeForUiTests", "-PideaVersion=" + ideaVersion.toString(), "-Drobot-server.port=" + port);
```

Comment:
both commands are almost the same. The path to executable can be fixed by System.getProperty("file.separator"); and then you just need to add iff for windows.

Reply:
Fixed.